mitwindfarm/Wake.py

```diff
@@ -155,7 +155,37 @@ def wake_added_turbulence(
         )
 
         return gaussian_ * np.nan_to_num(WATI)
+    
+    def RE_wake_added_turbulence(
+        self, x_glob: ArrayLike, y_glob: ArrayLike, z_glob=0
+    ) -> ArrayLike:
+        """
+        Following Niayifar and Porte-Agel 2016 and
+        Weisstein, Eric W. "Circle-Circle Intersection."
+
+        Returns wake added turbulence intensity caused by a wake at a particular
+        rotor by computing the overlap of a 4 sigma diameter (double wake width)
+        top hat with the downstream rotor. """
+         
+        x, y, z = x_glob - self.x, y_glob - self.y, z_glob - self.z
+        R = 2 * self.sigma * self._wake_diameter(x)
+        r = 0.5
+        d = np.sqrt(y ** 2 + z ** 2)
+
+        WATI = self.centerline_wake_added_turb(x)
 
+        with np.errstate(all = "ignore"):
+            wake_overlap = (
+                (r ** 2) * np.arccos((d ** 2 + r ** 2 - R ** 2) / (2 * d * r)) + 
+                (R ** 2) * np.arccos((d ** 2 + R ** 2 - r ** 2) / (2 * d * R)) -
+                0.5 * np.sqrt((-d + r + R)*(d + r - R)*(d - r + R)*(d + r + R))
+            )
+            wake_overlap[d > r + R] = 0
+            wake_overlap[d < R - r] = np.pi * r ** 2
+            return (wake_overlap * 4 * np.nan_to_num(WATI)) / np.pi
```

Comment:
`np.nan_to_num` isnt implemented in dualitic yet, but it probably would be quite straitght forward. On the other hand, we should consider if we want to convert nans to numbers at all as it could make it seem like the code is working when it actually isnt

Reply:
I have added because it is also used in wake_added_turbulence to process the direct output of wake_added_centerline_turbulence. Why is it used in wake_added_turbulence?